srk/src/transitionSystem.ml

```diff
@@ -662,18 +684,168 @@ module Make
     in
     List.map invariants (L.all_loops (L.loop_nest tg))
 
-  let simplify p tg =
+
+  let inline ?(depth=(-1)) tg entry (displayer: t -> unit) (assertion_map : 'x SrkUtil.Int.Map.t) = 
```

Comment:
Is displayer just for debugging inline?  If yes, we can omit this parameter now.